Mainboard/README.md

```diff
@@ -254,3 +254,28 @@ as "JST SH", and has a 1.0 mm pitch. The same connector is used for the Speakers
 | 2   | FAN_SPEED |
 | 3   | FAN_PWM   |
 | 4   | GND       |
+
+## RTC Battery Interface
+
+NOTE: The mainboard will charge this battery, so a rechargable ML1220 battery must be used.
+CR2032 is not okay because it cannot be charged.
```

Comment:
you will get explody batteries if you do that, maybe we should include some warnings after a little more research.